facebook/react-native
# fix(dev-middleware): allow inspector proxy to fetch sourcemaps on lan connections

## Summary:

The inspector proxy is inlining source maps on `Debugger.scriptParsed` CDP events. The inlining prevents Chrome DevTools from downloading this remotely, as that's not supported in newer versions anymore.

The current implementation locks this inlining mechanism to just `localhost` and/or `127.0.0.1` addresses, making it incompatible with LAN or tunnel device connections.

This PR removes that limitation to allow source map inlining on these LAN and tunnel connections.

## Changelog:



[GENERAL][FIXED] Allow Inspector proxy to inline source maps on LAN connections

## Test Plan:

- See added test
- Start Metro and connect a device over LAN, open the chrome devtools
